utils/model_utils.py

```
@@ -414,11 +414,9 @@
                 data_seed=data_seed,
             )
             trainer = Trainer(
-                model_init=lambda m=model_name, d=dataset: (
-                    AutoModelForSequenceClassification.from_pretrained(
-                        m,
-                        num_labels=len(d.unique("label")),
-                    )
+                model_init=lambda: AutoModelForSequenceClassification.from_pretrained(
+                    model_name,
```

Ruff (0.8.2)

Function definition does not bind loop variable `model_name`

(B023)